Testsuite: au.org.ala.volunteer.TaskServiceTests
Tests run: 1, Failures: 0, Errors: 1, Time elapsed: 0.295 sec
------------- Standard Output ---------------
--Output from testProjectCounts--
------------- ---------------- ---------------
------------- Standard Error -----------------
--Output from testProjectCounts--
------------- ---------------- ---------------

Testcase: testProjectCounts took 0.271 sec
	Caused an ERROR
Error creating bean with name 'taskService': Cannot create inner bean '(inner bean)' while setting bean property 'target'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name '(inner bean)#2': Instantiation of bean failed; nested exception is org.springframework.beans.factory.BeanDefinitionStoreException: Factory method [public java.lang.Object org.codehaus.groovy.grails.commons.AbstractGrailsClass.newInstance()] threw exception; nested exception is org.codehaus.groovy.grails.exceptions.NewInstanceCreationException: Could not create a new instance of class [au.org.ala.volunteer.TaskService]!
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'taskService': Cannot create inner bean '(inner bean)' while setting bean property 'target'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name '(inner bean)#2': Instantiation of bean failed; nested exception is org.springframework.beans.factory.BeanDefinitionStoreException: Factory method [public java.lang.Object org.codehaus.groovy.grails.commons.AbstractGrailsClass.newInstance()] threw exception; nested exception is org.codehaus.groovy.grails.exceptions.NewInstanceCreationException: Could not create a new instance of class [au.org.ala.volunteer.TaskService]!
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name '(inner bean)#2': Instantiation of bean failed; nested exception is org.springframework.beans.factory.BeanDefinitionStoreException: Factory method [public java.lang.Object org.codehaus.groovy.grails.commons.AbstractGrailsClass.newInstance()] threw exception; nested exception is org.codehaus.groovy.grails.exceptions.NewInstanceCreationException: Could not create a new instance of class [au.org.ala.volunteer.TaskService]!
	at _GrailsTest_groovy$_run_closure4.doCall(_GrailsTest_groovy:271)
	at _GrailsTest_groovy$_run_closure4.call(_GrailsTest_groovy)
	at _GrailsTest_groovy$_run_closure2.doCall(_GrailsTest_groovy:228)
	at _GrailsTest_groovy$_run_closure1_closure21.doCall(_GrailsTest_groovy:187)
	at _GrailsTest_groovy$_run_closure1.doCall(_GrailsTest_groovy:174)
	at TestApp$_run_closure1.doCall(TestApp.groovy:82)
	at gant.Gant$_dispatch_closure5.doCall(Gant.groovy:381)
	at gant.Gant$_dispatch_closure7.doCall(Gant.groovy:415)
	at gant.Gant$_dispatch_closure7.doCall(Gant.groovy)
	at gant.Gant.withBuildListeners(Gant.groovy:427)
	at gant.Gant.this$2$withBuildListeners(Gant.groovy)
	at gant.Gant$this$2$withBuildListeners.callCurrent(Unknown Source)
	at gant.Gant.dispatch(Gant.groovy:415)
	at gant.Gant.this$2$dispatch(Gant.groovy)
	at gant.Gant.invokeMethod(Gant.groovy)
	at gant.Gant.executeTargets(Gant.groovy:590)
	at gant.Gant.executeTargets(Gant.groovy:589)
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: Factory method [public java.lang.Object org.codehaus.groovy.grails.commons.AbstractGrailsClass.newInstance()] threw exception; nested exception is org.codehaus.groovy.grails.exceptions.NewInstanceCreationException: Could not create a new instance of class [au.org.ala.volunteer.TaskService]!
Caused by: org.codehaus.groovy.grails.exceptions.NewInstanceCreationException: Could not create a new instance of class [au.org.ala.volunteer.TaskService]!
Caused by: org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'au.org.ala.volunteer.TaskService$_closure1@4d89136d' with class 'au.org.ala.volunteer.TaskService$_closure1' to class 'java.util.Map'
	at au.org.ala.volunteer.TaskService.<init>(TaskService.groovy)

